actors/common/src/main/java/org/sunbird/learner/util/UserFlagEnum.java

```diff
@@ -0,0 +1,30 @@
+package org.sunbird.learner.util;
+
+public enum UserFlagEnum {
```

Comment:
Provide java doc to explain logic of enum.

Reply:
Done